pkgs/applications/editors/emacs/elisp-packages/manual-packages/beancount-mode/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  trivialBuild,
+  fetchFromGitHub,
+  emacs,
+  lib,
+}:
+trivialBuild rec {
+  pname = "beancount-mode";
+  version = "2023-02-04";
+
+  src = fetchFromGitHub {
+    owner = "beancount";
+    repo = "beancount-mode";
+    rev = "519bfd868f206ed2fc538a57cdb631c4fec3c93e";
+    hash = "sha256-nTEXJdPEPZpNm06uYvRxLuiOHmsiIgMLerd//dA0+KQ=";
+  };
+
+  meta = {
+    homepage = "https://github.com/beancount/beancount-mode";
+    description = "Emacs major-mode to work with Beancount ledger files ";
+    maintainers = with lib.maintainers; [polarmutex];
+    license = lib.licenses.gpl3;
```

Comment:
`gpl3` is , `gpl3Only` or `gpl3Plus` is preferred.

https://www.gnu.org/licenses/identify-licenses-clearly.html